elastic/kibana
# Distinguish not found SO errors for TM metrics

Resolves: #174749

## To verify:

Run Kibana and create a rule with actions.
Replace one of the getDecryptedAsInternalUser's (e.g. [getActionInfo](https://github.com/elastic/kibana/pull/174840/files#diff-9a1c0e513210895d1de20260558acd5fadacf4b8331879fba6f37a3e96375fabR473)) id input with a random number and check for task_run  user_errors on http://localhost:5601/api/task_manager/metrics?reset=false
